querydsl-core/src/main/java/com/querydsl/core/types/expr/Wildcard.java

```diff
@@ -36,12 +36,12 @@
     public static final NumberExpression<Long> count = NumberOperation.create(Long.class, Ops.AggOps.COUNT_ALL_AGG);
     
     /**
-     * Wilcard count distinct expression ((count(distinct *))
+     * Wildcard count distinct expression ((count(distinct *))
      */
     public static final NumberExpression<Long> countDistinct = NumberOperation.create(Long.class, Ops.AggOps.COUNT_DISTINCT_ALL_AGG);
 
     /**
-     * Wilcard count distinct expression ((count(distinct *))
+     * Wildcard count distinct expression ((count(distinct *))
      */
     public static final NumberExpression<Integer> countAsInt = NumberOperation.create(Integer.class, Ops.AggOps.COUNT_ALL_AGG);
```

Comment:
Good catch as well. I think it is desirable to wrap it in parentheses, so adding one on the right is good.